fix(ecs): compute_instance support update user_data (#4075)
huaweicloud/terraform-provider-huaweicloud: docs/resources/compute_instance.md

* `eip_id` - (Optional, String, ForceNew) Specifies the ID of an *existing* EIP assigned to the instance.
  This parameter and `eip_type`, `bandwidth` are alternative. Changing this creates a new instance.

> * `user_data` - (Optional, String) Specifies the user data to be injected during the instance creation. Text
>   and text files can be injected. This value support unencrypted, also support base64 encrypted.

Reviewer: ```suggestion
* `user_data` - (Optional, String) Specifies the user data to be injected to the instance during the creation. Text
  and text files can be injected. The content of `user_data` can be plaint text or encoded with base64.
```

Note that the imported state may not be identical to your resource definition, due to some attributes missing from the
API response, security or some other reason.

> The missing attributes include: `admin_pass`, `metadata`, `data_disks`, `scheduler_hints`, `stop_before_destroy`,

Reviewer 2: the value of the parameter `user_data` is never save in the Read function, why we remove it?

Author: I misunderstood, and have already corrected it back.
